### ISSUE TITLE
Peripheral dysostosis

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10224]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 23933701 Shortening of the tubular bones of the hands and feet with cone-shaped epiphyses is defined as **peripheral dysostosis** and is common to many syndromes including acrodysostosis and AHO or pseudohypoparathyroidism.

PMID: 36324970 Acrodysostosis (ACRO) is a disorder in peripheral bone development initially described in 1968 by Maroteaux. It is a member of the dysostoses, characterized by bone malformations attributed to blastogenesis abnormalities. It is a rare pathology, with <70 cases reported in the literature. Signature characteristics of the disease include: brachydactyly affecting all metacarpals, diffuse **peripheral dysostosis**, short stature, skull hypoplasia, maxillonasal hypoplasia, and shortened interpedicular distance due to premature closure of vertebral epiphyses, which contributes to spinal stenosis.

**Parent term (use hpo.jax.org/app)**
Abnormal digit morphology HP:0011297

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

acrodysostosis ORPHA:950
pseudohypoparathyroidism ORPHA:97593
Albright’s hereditary osteodystrophy ORPHA:665

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
I do not think that peripheral dysostosis is an appropriate HPO term.
Dysostosis is a general description for a localized developmental defect of bone.
Peripheral means it is affecting hands and feet.
However, it is not a precise description of what the actual defects are. For instance, in the first paper cited, Figure two shows "broad, short metacarpals and phalanges with cone-shaped epiphyses and brachydactyly". There are HPO terms for each of these features and saying "peripheral dysostosis" does not really add anything to the description.
I realize that articles sometimes use this phrase, but it is a kind of sloppy/bundled way of description that should be avoided in the future!
